Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/SharedRepositoryNavPage.xaml

```diff
@@ -33,23 +33,42 @@
                 </Style>
             </TabControl.Resources>
```

Comment:
also here i see you created duplication of existing SharedRepositoryPage we have so either remove the other one or use it instead